### PR TITLE
Adding image (jpg/png) output

### PR DESCRIPTION
**Pull request by ceb8**
This pull request adds the image output feature, and includes testing an documentation for the new functionality.

This includes two new public functions:
- `img_cut`
  Similar to `fits_cut` but returns the cutout(s) as either pngs or jpgs.
  Produces single band images as default, but can produce color images as well.
  Allows the image clipping/stretch to be specified (defaults to 99% clipping and asinh stretch)
- `normalize_img`
  Performs the image scaling/stretching.
  Input and output are arrays.

Additionally I fixed the bug reported in https://github.com/spacetelescope/astrocut/issues/20.

### CODE REVIEW

**Comment by phicc** [comment]
Comment

File: docs/astrocut/index.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ limitation:
 This operation can also take some time to run. For the 1348 FFI images of the TESS ete-6
 simulated sector, it takes about 12 minutes to run on a computer with 65 GB of memory.
 
-By default *make_cube* runs in verbose mode and prints out it's progress, however setting
+By default ``make_cube`` runs in verbose mode and prints out it's progress, however setting
```
Comment (jeffchamblee) [inline docs/astrocut/index.rst]: Should be "its" not "it's"

File: docs/astrocut/index.rst [hunk 2]
```diff
@@ -133,39 +133,94 @@ TESS FFIs are large and therefore are described by WCS objects that have many no
...
                 No.    Name      Ver    Type      Cards   Dimensions   Format
                   0  PRIMARY       1 PrimaryHDU      11   ()      
                   1  CUTOUT        1 ImageHDU        44   (200, 300)   float32   
                   2  CUTOUT        1 ImageHDU        44   (200, 300)   float32    
 
 
+The function `~astrocut.img_cut` takes one or more fits files and performs the same cutout in each,
+returning the result either an image (jpg or png) per cutout, or a single color image.
+It is important to remember that while the expectation is that all input images are
+aligned and have the same pixel scale, no checking is done.
```
Comment (jeffchamblee) [inline docs/astrocut/index.rst]: Would it be worthwhile to add checks for alignment and pixel scale to your code?

File: docs/astrocut/index.rst [hunk 3]
```diff
@@ -133,39 +133,94 @@ TESS FFIs are large and therefore are described by WCS objects that have many no
...
-or as one fits file per cutout. It is important to remember that while the expectation is
-that all input image are aligned and have the same pixel scale, no checking is done.
+These functions provide general purpose fits cutout functionality, returning the results either
+as fits cutout files, or as images (jpg or png).
+An image normalization (`~astrocut.normalize_img`) function is also available.
+
+The function `~astrocut.fits_cut` takes one or more fits files and performs the same cutout in each,
+returning the result either in a single fits file or as one fits file per cutout.
+It is important to remember that while the expectation is that all input image are
+aligned and have the same pixel scale, no checking is done.
```
Comment (jeffchamblee) [inline docs/astrocut/index.rst]: Same comment as below.

File: docs/astrocut/index.rst (hunk 2 above)
Comment (ceb8) [reply docs/astrocut/index.rst]: Yes, that would definitely be an improvement. It's not straightforward though, so probably warrants its own ticket/issue.

**Pull request merged**